src/constants/translations/en/my-resources-page.json

```diff
@@ -0,0 +1,8 @@
+{
+  "tabs": {
+    "courses": "Courses",
```

Comment:
Delete courses because now we don't have such a tab.

Reply:
fixed